vocabulary-draft/continuous-deployment-pipeline-events.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Deployment Pipelines Events 
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+These events are related to continous deployment pipelines and their target environments. 
+These events can be emitted by environments to report where software artifacts such as services, binaries, deamons, jobs or embedded software are running. 
+
+The term Service is used to represent a running Artifact. This service can represent a binary that is running, a deamon, an application, a docker container, etc.
+The term Environment represent any platform which has all the means to run a Service. 
+
+- **Environment Created**: an environment has been created and it can be used to deploy Services
+- **Environment Modified**: an environment has been modified, this event advertise the changes made in the environment
+- **Environment Deleted**: an environment has been deleted and cannot longer be used
```

Comment:
An environment is something like "test", "stage" or "production"? 
A kubernets cluster or a VM?

Reply:
@afrittoli all of the above..  it is a name where services are deployed